packages/examples/import/STLImport/index.js

```diff
@@ -0,0 +1,24 @@
+/*
+// title       : STL Import Demonstration
+// author      : Simon Clark
+// license     : MIT License
+// description : Importing STL files to manipulate and combine. Drag the whole STLImport folder into JSCAD
+// file        : STLImport/index.js
+// tags        : stl, import
+*/
+
+const { translate, scale, rotateZ } = require('@jscad/modeling').transforms
+const { union } = require('@jscad/modeling').booleans
+
+// Load the STL files using require
+const sculpture = require('./3d_sculpture-VernonBussler.stl')
```

Comment:
what do you think about the STL / AMF / etc files in the examples? i think we could find some low poly 'modern' examples.

Reply:
I don't think there's a big issue to have the example files we have.  What do you think we gain be different examples.